### PR TITLE
new tau vlaues

### PR DESCRIPTION
**Pull request by DouglasBurns**

### CODE REVIEW

File: config/cross_section_config.py [hunk 1]
```diff
@@ -672,13 +672,13 @@ def __fill_defaults_13TeV__( self ):
...
-"NJets" : 0.932603346883,
+"MET" : 2.15443469003,
+"WPT" : 1.12332403298,
+"NJets" : 1.12332403298,
 "abs_lepton_eta" : 0.1,
-"ST" : 0.932603346883,
+"ST" : 0.642807311728,
 "lepton_pt" : 0.585702081806,
-"HT" : 1.12332403298,
+"HT" : 0.585702081806,
```
Comment (EmyrClement) [inline config/cross_section_config.py]: Are the tau values for HT and lepton pt really identical?  Same for WPT and NJets?

**Comment by DouglasBurns** [comment]
Not sure why its appended it self the this request. (bad) here just means my current binning which is too fine. Better binning by increasing nmin to 500, 3000 respectively is commented out

**Comment by EmyrClement** [comment]
I don't really understand the comment?

File: config/variable_binning.py [hunk 2]
```diff
@@ -5,24 +5,38 @@
...
+'lepton_pt' : [23.0, 25.0, 27.0, 29.0, 31.0, 33.0, 35.0, 37.0, 39.0, 41.0, 43.0, 45.0, 47.0, 49.0, 51.0, 53.0, 55.0, 57.0, 59.0, 61.0, 63.0, 66.0, 69.0, 72.0, 75.0, 78.0, 81.0, 84.0, 87.0, 90.0, 94.0, 98.0, 102.0, 106.0, 110.0, 114.0, 119.0, 125.0, 131.0, 139.0, 149.0, 163.0, 184.0, 268.0],
+'abs_lepton_eta' : [0.0, 0.018, 0.036000000000000004, 0.054000000000000006, 0.07200000000000001, 0.09000000000000001, 0.108, 0.126, 0.14400000000000002, 0.162, 0.18, 0.198, 0.216, 0.234, 0.252, 0.27, 0.28800000000000003, 0.306, 0.324, 0.342, 0.36, 0.378, 0.396, 0.41400000000000003, 0.432, 0.45, 0.468, 0.486, 0.504, 0.522, 0.54, 0.558, 0.5760000000000001, 0.594, 0.615, 0.633, 0.654, 0.675, 0.6960000000000001, 0.714, 0.735, 0.756, 0.777, 0.798, 0.8190000000000001, 0.84, 0.861, 0.882, 0.906, 0.927, 0.9510000000000001, 0.975, 0.999, 1.023, 1.047, 1.071, 1.095, 1.1219999999999999, 1.149, 1.176, 1.2029999999999998, 1.23, 1.257, 1.2839999999999998, 1.3139999999999998, 1.3439999999999999, 1.3739999999999999, 1.404, 1.4369999999999998, 1.4789999999999999, 1.53, 1.5779999999999998, 1.617, 1.656, 1.698, 1.74, 1.785, 1.8359999999999999, 1.884, 1.938, 1.9949999999999999, 2.223],
```
Comment (kreczko) [inline config/variable_binning.py]: Personally I would edit the bins by hand to smooth them out.
Judging by the first few bins, the size seems to be 0.018

**Comment by DouglasBurns** [comment]
The current binnings are for what I showed in the meeting. The commented
out bins are me messing around with NEventMin to get a better binning.
YThis was 500 for non leptonic variables and 3000 for leptonic variables

*________________________*


*Douglas Burns*
*PhD Student*

School of Physics
University of Bristol
HH Wills Physics Lab
University of Bristol
Tyndall Avenue
Bristol
BS8 1TL

Douglas.Burns@bristol.ac.uk

File: config/variable_binning.py (hunk 2 above)
Comment (DouglasBurns) [inline config/variable_binning.py]: Sorry yeah with the binnings I did that the first time round.  I wasnt
really planning on adding to a pull request this quickly so havent bothered
altering the readability yet. (I will smoth them out)

*________________________*


*Douglas Burns*
*PhD Student*

School of Physics
University of Bristol
HH Wills Physics Lab
University of Bristol
Tyndall Avenue
Bristol
BS8 1TL

Douglas.Burns@bristol.ac.uk

**Comment by EmyrClement** [comment]
Is this request still relevant?  Should it be merged?

**Comment by EmyrClement** [comment]
Is this request still relevant? Should it be merged or closed?

**Comment by DouglasBurns** [comment]
It has relevant bits in it, though id like to split it up into smaller PRs
somehow, if thats possible? It can be closed for now at any rate

*________________________*


*Douglas Burns*
*PhD Student*

School of Physics
University of Bristol
HH Wills Physics Lab
University of Bristol
Tyndall Avenue
Bristol
BS8 1TL

Douglas.Burns@bristol.ac.uk

**Comment by kreczko** [comment]
Of course it is possible :), but requires some effort.
You can cherry-pick any commit.

**Pull request closed without merging**